packages/payload/src/bin/migrate.ts

```diff
@@ -32,7 +32,27 @@ type Args = {
 }
 
 export const migrate = async ({ config, parsedArgs }: Args): Promise<void> => {
-  const { _: args, file, forceAcceptWarning, help } = parsedArgs
+  const { _: args, file, forceAcceptWarning: forceAcceptFromProps, help } = parsedArgs
+
+  const formattedArgs = Object.keys(parsedArgs || {})
+    .map((key) => {
+      const formattedKey = key.replace(/^[-_]+/, '')
+      if (!formattedKey) {
+        return null
+      }
+
+      return formattedKey
+        .split('-')
+        .map((word, index) =>
+          index === 0 ? word.toLowerCase() : word.charAt(0).toUpperCase() + word.slice(1),
+        )
+        .join('')
+    })
+    .filter(Boolean)
+
+  const forceAcceptWarning = forceAcceptFromProps || formattedArgs?.includes('forceAcceptWarning')
+  const skipVerify = formattedArgs?.includes('skipVerify')
```

Comment:
Need to handle `skipEmpty` here as well.

Reply:
Updated